include/dfrn.php

```diff
@@ -1484,8 +1608,10 @@ private static function process_suggestion($xpath, $suggestion, $importer) {
 			dbesc(normalise_link($suggest["url"])),
 			intval($suggest["uid"])
 		);
-		if (dbm::is_result($r))
+		/// @TODO Really abort on valid result??? Maybe missed ! here?
```

Comment:
The valid result means the friend we're about to send a friend suggestion already has them in their contact, which means no further action is required. Thanks for your vigilance.